src/atomizer.js

```diff
@@ -141,18 +147,28 @@ Atomizer.prototype.parseConfig = function (config/*:AtomizerConfig*/, options/*:
         var rgb;
         var values;
 
-        if (!match) {
-          return '';
+        if (!match || (!match.atomicSelector && !match.selector)) {
+          // no match, no op
+          return;
         }
 
-        ruleIndex = match.prop ? this.rulesMap[match.prop] : this.helpersMap[match.helper];
+        // check where this rule belongs to
+        if (this.rulesMap.hasOwnProperty(match.atomicSelector)) {
+            ruleIndex = this.rulesMap[match.atomicSelector];
+        } else if (this.helpersMap.hasOwnProperty(match.atomicSelector)) {
+            ruleIndex = this.helpersMap[match.atomicSelector];
+        } else if (this.helpersMap.hasOwnProperty(match.selector)) {
+            ruleIndex = this.helpersMap[match.selector];
+        } else {
+            // not a valid class, no op
+            return;
+        }
```

Comment:
Add more docs here would be helpful, like what is an `atomicSelector` and `selector`